schemas/productSource.schema.tpl.json

```diff
@@ -6,27 +6,27 @@
   ],
   "properties": {
     "digitalIdentifier": {
-      "_instruction": "Add the globally unique identifier for this product (e.g. 'Research Resource Identifier' (RRID)).",
+      "_instruction": "Add the globally unique and persistent digital identifier of this product.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/core/RRID"
       ]
     },
     "identifier": {
-      "_instruction": "Enter one or several identifiers for this product excluding globally unique identifiers such as RRIDs. e.g. catalog number",
-      "type": "string"
+      "type": "string",
+      "_instruction": "Enter the identifier for this product, excluding its RRID (e.g., a catalog number)."
     },
     "productName": {
       "type": "string",
-      "_instruction": "Enter the name of the product as provided by the vendor."
+      "_instruction": "Enter the name of this product as stated by the 'provider'."
     },
     "provider": {
-      "_instruction": "Add the provider of the product.",
+      "_instruction": "Add the party (private, commercial or industrial) that provided this product.",
```

Comment:
It seems good enough, just a suggestion to make it better.
 `Add the party (whether it be private, commercial, or industrial) that provided this product.`

Reply:
I don't think that this is necessary. What do @apdavison & @lzehl think? If you prefer @Peyman-N suggestion, I will change it 🙂

Reply:
I agree, lets resolve it for the moment :)